spectacles/lookml.py

```diff
@@ -322,14 +325,46 @@ def get_explore(self, model: str, name: str) -> Optional[Explore]:
             return model_object.get_explore(name)
 
     def get_results(
-        self, validator: str, fail_fast: Optional[bool] = None
+        self,
+        validator: str,
+        fail_fast: Optional[bool] = None,
+        filters: Optional[List[str]] = None,
     ) -> Dict[str, Any]:
         errors: List[Dict[str, Any]] = []
         successes: List[Dict[str, Any]] = []
         tested = []
 
         for model in self.models:
+            # Add model level content validation errors.
+            # We create an explore "tested" record for those that
+            # aren't in the LookML tree.
+            distinct_explores = set()
+            if filters is None and model.errors:
+                # Raise name error if there are errors on the model and filters
+                # isn't being passed. We need filters when running the content
+                # validator, which is the only time a model will have errors.
+                raise NameError(
+                    "The argument 'filters' is required in Content Validator."
```

Comment:
Also, the more I think about this, from the standpoint of `get_results`, I don't see a reason why you couldn't ask for results with model errors and no filters. I think probably it's better for us to validate this a layer up when it's called in the Content Validator itself rather than here.

Reply:
I think there's possibly a few issues with that approach, but I could be wrong:

* Even if in the content validator we only attached 'selected' errors to the tree, we would still be returning the whole tree. Then, when we call `get_results`, you would end up with records in `tested` for explores that should not be tested as far as the end-user is concerned. 
* If we wanted resolve the issue in the point above, we would need to prune the tree in the content validator, which doesn't really feel like something that is in it's domain, but if we're happy to do that, we could definitely move it there.